clang/lib/AST/ItaniumMangle.cpp

```diff
@@ -2456,67 +2456,6 @@ static bool isTypeSubstitutable(Qualifiers Quals, const Type *Ty,
   return true;
 }
 
-namespace {
```

Comment:
That's interesting, that we had no Half handling for Microsoft mangler.

Reply:
We had and still have it, but this bunch of code was invented to mangle `cl::sycl::detail::half_impl::half` in the same manner as `_Float16`